src/api/apiEndpoint.ts

```diff
@@ -2,6 +2,18 @@ import URI from "urijs";
 import storage from "../storage";
 
 export default function apiEndpoint() {
+  if (window.getComitNodeApiEndpoint) {
```

Comment:
:mega: Correct me if I'm wrong, but it now looks like you won't be able to change host and port through the settings page if comit-i is served by the comit node. I would instead save these values provided by the comit node on comit-i startup to local storage and keep this function untouched so that they can still be modified by the user.

Reply:
That is the intended behaviour at the moment yes. When comit-i is served by the comit_node, there is not really a point in the user changing those settings since it will render comit-i unusable.

It may be worth mentioning that on the settings page and disable the form if that is the case.

Reply:
@thomaseizinger are you saying we should disable the form now? I suggest that yesterday and proposed to keep the solution as simple as possible and let users open a bug if needed...

Reply:
@D4nte We could yes.

I don't think we should temper with the values the user saved into local storage. Depends on how keen you are to do some React @D4nte you could disable the form + display a message if the "override" is active :)

Reply:
I guess it makes sense to only allow the embedded comit-i to connect to the comit node that serves it, but in that case I would disable the fields in the settings page. But that can be done in a separate issue if it's very low priority. Do open an issue though :smile: